worlds/hk/docs/setup_en.md

```diff
@@ -44,9 +46,7 @@ website to generate a YAML using a graphical interface.
    * If you are waiting for a countdown then wait for it to lapse before hitting Start.
    * Or hit Start then pause the game once you're in it.
    
-## Commands
-While playing the multiworld you can interact with the server using various commands listed in the 
-[commands guide](/tutorial/Archipelago/commands/en). As this game does not have an in-game text client at the moment,
-You can optionally connect to the multiworld using the text client, which can be found in the 
-[main Archipelago installation](https://github.com/ArchipelagoMW/Archipelago/releases) as Archipelago Text Client to
-enter these commands.
+## Hints and other commands
+While playing in a multiworld, you can interact with the server using various commands listed in the 
+[commands guide](/tutorial/Archipelago/commands/en). In order to do this, install the latest release of the [Archipelago software](https://github.com/ArchipelagoMW/Archipelago/releases/latest)
```

Comment:
The sentence goes "in order to do this [...] to enter commands". You could probably just remove "to enter commands" entirely

Reply:
I felt like that phrasing was weird. This sentence is essentially "you want to enter commands, here is how to do that. Step 1, install the archipelago software. Step 2, open the text client and that is where you enter the commands." But more streamlined

Reply:
Maybe "you can do this by launching the archipelago text client, included in the latest release of the archipelago software" would be better?